cmd/bootstrap/run/result.go

```diff
@@ -5,12 +5,19 @@ import (
 )
 
 func GenerateRootResult(block *flow.Block, commit flow.StateCommitment) *flow.ExecutionResult {
+	// add commit as endstate of chunk
+	chunks := flow.ChunkList{}
+	chunk := &flow.Chunk{
+		Index:    0,
+		EndState: commit,
+	}
+	chunks.Insert(chunk)
```

Comment:
can we extract this into a function and move it to `model/flow/chunk.go`, so that the implementation details are encapsulated.

```
func ChunListFromOneCommit(commit flow.StateCommitment) flow.ChunkList
```

Could also reuse the function [here](https://github.com/onflow/flow-go/pull/25/files#diff-cd3f8a4b0303d678482283c9bb6f0707R824-R829)

Reply:
Ahh yes, ok implemented! Thanks.